singer_sdk/sql/connector.py

```diff
@@ -0,0 +1,914 @@
+"""Base class for SQLAlchemy-based connectors."""
```

Comment:
@kgpayne - Any changes to call out in the moving of the `SQLConnector` class to the new module?

From the diff, we can't really tell if nothing is changed or if a lot has changed.

Reply:
There were more changes than was simple to call out, so I undid the new module. I still think moving the `SQLConnector` out to a file or module would make it easier to work on, but that can be done in a future PR with no other changes 🙂